aspnetcore/data/ef-rp/read-related-data.md

```diff
@@ -31,57 +32,87 @@ The following illustrations show the completed pages for this tutorial:
 
-  ![Explicit loading example](read-related-data/_static/explicit-loading.png)
+ ![Explicit loading example](read-related-data/_static/explicit-loading.png)
 
-* Lazy loading. When the entity is first read, related data isn't retrieved. However, the first time you attempt to access a navigation property, the data required for that navigation property is automatically retrieved. A query is sent to the DB each time you try to get data from a navigation property for the first time. Entity Framework Core 1.0 does not support lazy loading.
+* [Lazy loading](https://docs.microsoft.com/ef/core/querying/related-data#lazy-loading). [EF Core does not currently support lazy loading](https://github.com/aspnet/EntityFrameworkCore/issues/3797). When the entity is first read, related data isn't retrieved. The first time a navigation property is accessed, the data required for that navigation property is automatically retrieved. A query is sent to the DB each time a navigation property is accessed for the first time.
+
+* The `Select` operator loads only the related data needed, and is generally faster than using `Include`. [Loading related data with Select](#select) explains this in more detail.
 
 ### Performance considerations
 
-If you know you need related data for every entity retrieved, eager loading often offers the best performance, because a single query sent to the DB is typically more efficient than separate queries for each entity retrieved. For example, suppose that each department has ten related courses. Eager loading of all related data would result in just a single (join) query and a single round trip to the DB. A separate query for courses for each department would result in eleven round trips to the DB. The extra round trips to the DB are especially detrimental to performance when latency is high.
+If related data for every entity retrieved is needed:
 
-On the other hand, in some scenarios separate queries is more efficient. Eager loading of all related data in one query might cause a very complex join to be generated, which SQL Server can't process efficiently. Or if you need to access an entity's navigation properties only for a subset of a set of the entities you're processing, separate queries might perform better because eager loading of everything up front would retrieve more data than you need. If performance is critical, it's best to test performance both ways in order to make the best choice.
+* Eager loading generally offers the best performance.
+* Eager loading sends a single query to the DB. A single query is typically more efficient than separate queries for each entity retrieved.
```

Comment:
See comment above

Reply:
Eager loading sends multiple queries any time a collection nav is being included - one query for the main query and one query of each collection "edge" in the load tree.